core/frontend/src/components/vehiclesetup/configuration/failsafes/Failsafes.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div v-if="params_finished_loaded" class="pt-1 pb-1">
+    <v-sheet class="ma-3 pa-1 d-flex flex-row flex-wrap flex-grow-0 justify-center">
+      <failsafe-card
+        v-for="failsafeDefinition in failsafes"
+        :key="failsafeDefinition.name"
+        :failsafe-definition="failsafeDefinition"
+      />
+    </v-sheet>
+  </div>
+</template>
+
+<script lang="ts">
+import failsafeCard from '@/components/vehiclesetup/configuration/failsafes/FailsafeCard.vue'
+import autopilot_data from '@/store/autopilot'
+
+import { FailsafeDefinition } from './types'
+
+export default {
+  name: 'FailsafesConfigration',
+  components: {
+    failsafeCard,
+  },
+  data() {
+    return {
+      failsafes: [] as FailsafeDefinition[],
+    }
+  },
+  computed: {
+    params_finished_loaded(): boolean {
+      return autopilot_data.finished_loading
+    },
+  },
+  mounted() {
+    // we fetch the failsafes on mounting because we
+    // need to be in an async context to wait for the import calls
+    this.get_failsafes().then((failsafes) => {
+      this.failsafes = failsafes
+    })
+  },
+  methods: {
+    async get_failsafes(): Promise<FailsafeDefinition[]> {
+      const failsafes: FailsafeDefinition[] = [
+        {
+          name: 'Ground Station Heartbeat Loss',
+          generalDescription: 'The heartbeat failsafe is triggered when the vehicle does not receive '
+          + 'a heartbeat from the GCS within 3 seconds.',
```

Comment:
```suggestion
          generalDescription: 'Triggers when the vehicle does not receive a heartbeat from the GCS '
          + 'within the timeout (default 3 seconds).',
```
The name already says what it's for. Given the active tone in the name we should continue that in the description.

Not sure if we can somehow make the end dynamically determined, where it says "the timeout" if the parameter exists, or "3 seconds" if the parameter doesn't exist. If not then I think this suggestion is an ok compromise.

Reply:
I think it can be done, but I'd rather keep this PR shorter. we could have a field to override the description text that uses a computed property 🤔